src/components/Cluster/ClusterDetail/AddNodePool.js

```diff
@@ -352,8 +446,48 @@ class AddNodePool extends Component {
               value={this.state.aws.instanceType.value}
             />
             <p>{`${CPUCores} CPU cores, ${RAM} GB RAM each`}</p>
+            {this.state.allowAlikeInstances && (
+              <FlexWrapperDiv className='alike-wrapper'>
+                <CheckboxWrapper>
+                  <input
+                    className='original-checkbox'
+                    type='checkbox'
+                    checked={this.state.aws.useAlike}
+                    onChange={this.setUseAlikeInstancesEnabled}
+                    id='aws-alike-instances-enabled'
+                  />
+                </CheckboxWrapper>
+                <label htmlFor='aws-alike-instances-enabled'>
+                  <p>Allow usage of similar instance types</p>
+                </label>
+              </FlexWrapperDiv>
+            )}
           </FlexWrapperDiv>
         </label>
+        {this.state.allowSpotInstances && (
+          <SpotValuesLabel>
+            <span className='label-span'>Instance distribution</span>
+            <NumberPicker
```

Comment:
This has no maximum value, and if I try to use a large enough number, the node pool creation fails

Reply:
how large?

@ubergesundheit we can probably limit this to 1000 for now.

Reply:
so above `32767`?

Reply:
I tried `999999999999999999999999999999` 😬